pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/forward/FixedByteSVMutableForwardIndex.java

```diff
@@ -84,6 +100,19 @@ public FixedByteSVMutableForwardIndex(boolean dictionaryEncoded, DataType stored
     _chunkSizeInBytes = (long) numRowsPerChunk * _valueSizeInBytes;
     _memoryManager = memoryManager;
     _allocationContext = allocationContext;
+
+    // Initialize dynamic buffer sizing parameters from system properties
+    _dynamicBufferSizingEnabled = Boolean.parseBoolean(
+        System.getProperty(DYNAMIC_BUFFER_SIZING_ENABLED_KEY,
```

Comment:
Pinot doesn't rely on system properties. These should be introduced as forward index config in table config